src/main/java/com/findmyrecycling/fmrenterprise/dao/FacilityDAOStub.java

```diff
@@ -26,10 +26,9 @@ public Facility save(Facility facility) {
 
     @Override
     public List<Facility> fetchAll() {
-        return facilities;
+        return null;
```

Comment:
Hmmmm.   I'd rather return an empty list than null.  Null can cause a NullPointerException.
Even better, just implement it.  :)